run_fal.py

```diff
@@ -35,27 +36,30 @@
 parser.add_argument("--team-stats", action="store_true")
 parser.add_argument("--team-dist", action="store_true")
 parser.add_argument("--anime-weekly-stats", action="store_true")
+parser.add_argument("--team-score", action="store_true")
 parser.add_argument("--season", default=season_str)
 parser.add_argument("--year", default=year)
 args = parser.parse_args()
 
 
 if args.collect_series:
     collect_series()
-elif args.ptw_counter:
+if args.ptw_counter:
```

Comment:
Why are we changing to `if`? Should we allow different functions to run in succession?

Reply:
At the end of the day the end user is in control, and I think giving the user the option to run more than one feature could be useful, unless you think there's any risks?

Reply:
I was thinking it's probably safer to run each step at a time and make sure each step has worked before running the next the step, but I guess that could get tedious. We can use `if`.